config/locales/devise.en.yml

```diff
@@ -57,6 +57,8 @@ en:
   errors:
     messages:
       already_confirmed: "has already been confirmed. Sign in."
+      blank: "can’t be blank"
+      confirmation: "doesn’t match new password"
```

Comment:
This string assumes that `confirmation` only applies to passwords. Is this the correct assumption?

Reply:
I think so yes, we're not currently using the confirmable module though so I don't think this message actually ever shows anyway

Reply:
It shows when you change your password, and use 2 different values (that was how I came across the rogue straight quote!)

Reply:
Ah right, you've added this. Not sure if it's a strictly safe assumption for ever but it is the only thing in Devise using a`validates_confirmation_of` validation so should be fine.

https://github.com/heartcombo/devise/blob/main/lib/devise/models/validatable.rb#L41